pageserver/src/tenant.rs

```diff
@@ -509,6 +509,9 @@ pub enum TimelineArchivalError {
     #[error("Timeout")]
     Timeout,
 
+    #[error("HasArchivedParent")]
+    HasArchivedParent(TimelineId),
```

Comment:
```suggestion
    #[error("HasArchivedParent")]
    HasArchivedParent(TimelineId),
```

why not give out long form explanations for these instead of using `std::fmt::Debug` alike semi-literal strings?

Like this should/could have:

```suggestion
    #[error("parent is archived: {}", .0)]
    HasArchivedParent(TimelineId),
```